lyft_dataset_sdk/lyftdataset.py

```diff
@@ -97,14 +97,14 @@ def __init__(self, data_path: str, json_path: str, verbose: bool = True, map_res
 
     def __load_table__(self, table_name, verbose=False, missing_ok=False) -> dict:
         """Loads a table."""
-        filepath = str(self.json_path.joinpath("{}.json".format(table_name)))
+        filepath = self.json_path.joinpath("{}.json".format(table_name))
```

Comment:
Consider using f"{table_name}.json" instead of "{}.json".format(table_name)" Looks much more cleaner

Reply:
For sure. But cuts off users of python 3.5

Not ready for this yet.